CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Gas values in fuzzing test output are now displayed as whole numbers without fractional parts
 
+### Cast
+
+#### Fixed
+
+- A bug that prevented the `#[test_case]` attribute from being used on its own with contracts.
```

Comment:
Let's say that we fixed using `test_case` attribute with cheatcodes, this is more descriptive from the user standpoint.